src/Backpack.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Help me understand... what will this facade facilitate - what kind of calls? I'm wondering whether we need a Facade at all... or whether we should change its name.

Reply:
Is it to do stuff like `Backpack::getActiveController` and `Backpack::getCrud()`? If so... I think we're better of doing `CrudManager::getActiveController()` and `CrudManager::getCrud()`.

I wouldn't expand this feature so much as to identify it with the Backpack name. It would prevent us from using the Backpack class for anything else in the future.